apps/odyssey-stats/src/styles/wp-admin.scss

```diff
@@ -55,14 +55,6 @@
 		padding-top: 32px;
 		padding-left: 1px;
 	}
-	// Fixed header doesn't work well for WP-Admin, as the masterbar isn't fixed.
```

Comment:
@lupus2k What's the best way to test this change? 🤔

Reply:
We can rollback this change, I tried to remove all the fixed-navigation-header related but this one doesn't make sense.